src/main/java/org/broadinstitute/hellbender/engine/AbstractConcordanceWalker.java

```diff
@@ -85,10 +81,10 @@ private void initializeTruthVariantsIfNecessary() {
         return vc -> !vc.isFiltered();
     }
 
-    private Spliterator<TruthVersusEval> getSpliteratorForDrivingVariants() {
+    private Iterator<TruthVersusEval> getIteratorForDrivingVariants() {
```

Comment:
You can parallelize a `Spliterator`, but we weren't traversing in parallel, so this is okay.